### DISCUSSION TITLE
How to manage community issues

### DISCUSSION THREAD

**Discussion by JosephTLyons** [zed-industries/zed#8093] [Feature Requests]
We're looking for ways to improve our issue management. It is getting to be a bit of a heavy task to comb through and organize issues every few days, and it will only get more and more time-consuming as time goes on. I do want to read everything, but I'd like to have to do less tagging. Right now, my only thoughts are:

- Make more issue templates to cover more common cases that have the correct tags on things. The caveat here is that there will end up being quite a lot of templates, with duplicate content that will all need to be updated when we make changes to them.
- Find a way to generalize things more so less labeling is needed
- Look for open source maintainers. This is probably necessary anyway.
- Investigate tools to automate tagging (I don't know if any exist)

Keep in mind that we use the [core labels](https://github.com/zed-industries/zed/blob/2ec910f77240e699cc2f012088b220001d45b329/script/update_top_ranking_issues/main.py#L17) to generate our [top-ranking issue reports](https://github.com/zed-industries/zed/issues/6953)

Curious to hear if anyone has any suggestions for this?

**Comment by Moshyfawn** [comment]
A few things I've noticed while managing issues across Zed's repositories:

The current knowledgebase is spread across Discord, GitHub, and even Zed's in-editor chats. Part of this is due to the nature of the questions, but it's also due to some aspects of the docs being missing or not easily discoverable _(e.g. https://github.com/zed-industries/zed/discussions/8219#discussioncomment-8579374)_. I'm not saying the docs are bad, but rather there are a few gray areas that could be improved, like the ones mentioned above.

Apart from trying to improve the docs, I think moving away from Discord for support and towards GitHub Discussions would be a good move. It would allow for more structured conversations and make it easier to find answers to common questions and link to them in issues/duplicate questions. I know there has been an attempt to shut down the Discord questions, but people still ask questions there and I think it should be shut down completely in favour of GitHub Discussions.

More eyes (and hands) on deck would also help. Right now, answering questions and directing people to the right place is only helpful if you know the answer as an individual. I think additional maintainers with the ability to tag questions would help organise the current workload for better discoverability, where if you have time to answer questions, you can easily find the ones that have not yet been answered and are in your area of expertise, rather than having to go through all of them.

While I think a few more templates that, for example, point to a more general discussion/support question would be helpful, I think the current templates are good enough. I honestly feel that it's mostly a matter of getting people to use them, and then getting more maintainers to tag them.

On the subject of tagging, as far as I've been able to find out, I don't think there are really any tools to automate tagging. At this point, I think it's more about getting more maintainers to help with the workload as the community grows.
